SPECS/hyperscan/hyperscan.spec

```diff
@@ -64,7 +64,7 @@ needed for developing Hyperscan applications.
 # multiply defined symbols.  This seems like a GCC bug
 # Disable LTO
 %define _lto_cflags %{nil}
-%cmake -DBUILD_SHARED_LIBS:BOOL=ON -DBUILD_STATIC_AND_SHARED:BOOL=OFF .
+%cmake -DBUILD_SHARED_LIBS:BOOL=ON -DBUILD_STATIC_AND_SHARED:BOOL=OFF -DPYTHON_EXECUTABLE=/usr/bin/python3 .
```

Comment:
Minor. I'd suggest using the `%python3` just in case the default path ever changes:
```suggestion
%cmake -DBUILD_SHARED_LIBS:BOOL=ON -DBUILD_STATIC_AND_SHARED:BOOL=OFF -DPYTHON_EXECUTABLE=%{python3} .
```

The macro is defined inside `SPECS/mariner-rpm-macros/macros.python-srpm` in case you're curious.

Reply:
+1 to Pawel here.

Reply:
But is it really necessary to pass this at all?

Reply:
It seems necessary since I was hitting the following break without it: `Could NOT find PythonInterp (missing: PYTHON_EXECUTABLE) `

Reply:
switched to use the macro